Libplanet.Explorer/Mutations/TransactionMutation.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using GraphQL;
+using GraphQL.Types;
+using Libplanet.Action;
+using Libplanet.Blockchain;
+using Libplanet.Explorer.GraphTypes;
+using Libplanet.Explorer.Interfaces;
+using Libplanet.Tx;
+
+namespace Libplanet.Explorer.Mutations
+{
+    public class TransactionMutation<T> : ObjectGraphType
+        where T : IAction, new()
+    {
+        private readonly IBlockChainContext<T> _context;
+
+        public TransactionMutation(IBlockChainContext<T> context)
+        {
+            _context = context ?? throw new ArgumentNullException(nameof(context));
+
+            Field<TransactionType<T>>(
```

Comment:
When I looked the [`BlockChain<T>.StageTransaction(...)`](https://github.com/planetarium/libplanet/blob/6e0b23883040d3acfe5f3679f146b3d10e6ef587/Libplanet/Blockchain/BlockChain.cs#L653-L667), it seems return boolean value. Or it should return the `tx` variable if you intended to return the transaction.

```suggestion
            Field<BooleanGraphType>(
```

Reply:
Ah, thanks for pointing out. originally, I've intended returning `tx` but on second thought, returning `bool` is better.

Reply:
Hmm, I'll change to returning `tx` and throwing exception when `BlockChain<T>.StageTransaction()` returns `false`.